alurapic/src/app/core/auth/request.interceptor.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { TestBed } from '@angular/core/testing';
+
+import { HttpClient, HTTP_INTERCEPTORS, HttpResponse } from '@angular/common/http';
+import { HttpTestingController, HttpClientTestingModule } from '@angular/common/http/testing';
+import { RequestInterceptor } from './request.interceptor';
+import { TokenService } from '../token/token.service';
+
+describe('O interceptor RequestInterceptor', () => {
+  let service: RequestInterceptor;
+  let http: HttpClient;
+  let httpMock: HttpTestingController;
+  let tokenService: TokenService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [
+        RequestInterceptor,
+        TokenService,
+        {
+          provide: HTTP_INTERCEPTORS,
+          useClass: RequestInterceptor,
+          multi: true
+        }
+      ]
+    });
+    service = TestBed.get(RequestInterceptor);
+    http = TestBed.get(HttpClient);
+    httpMock = TestBed.get(HttpTestingController);
+    tokenService = TestBed.get(TokenService);
+  });
+
+  it('deve ser instaciada', () => {
+    expect(service).toBeTruthy();
+  });
+
+  describe('ao fazer requisições http', () => {
+    it('adiciona o token no header da requisição', () => {
```

Comment:
Acho legal deixar um padrão, já que usou o should nos outro :P 
"deve adicionar..."

Reply:
Não entendi essa observação